public/js/thirdpartyservices.js

```diff
@@ -102,5 +102,15 @@ var thirdPartyServicesJsonArray = [
       privacy: 'https://privacy.oath.com',
       terms: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html'
     }
+      , // Donot forget to add a ',' before adding your new item in the array
+  {
+    name: 'Fathom Analytics',
+    model: 'fathom', // this is stores the name of the model
+    gps: true, // this stores the state of the model, notice the key is the same as the model above. It is required by code.
```

Comment:
`gps` should be `fathom`. Read the comment right at this line.

Reply:
Done